database.c

```diff
@@ -362,7 +366,7 @@ int database_consumer_init(const char *ignore, const char *host, int port, const
 	return 1;
 }
 
-void database_consumer_finish() {
+void database_consumer_finish(int dry_run) {
```

Comment:
If you change the signature of `database_consumer_finish`, you also have to change the `typedef replay_item_consumer_finish` in `pgreplay.h`.

Are there no warnings and errors that the function signature is different from the prototype in `pgreplay.h`?

But as I said above, I'd just omit that.

Reply:
did not see any warning but i'll change the typedef of ```replay_item_consumer_finish``` nonetheless